README.md

```diff
@@ -1,3 +1,6 @@
+# TODO:
+[ ] fix styles manually via `composer cs-check`
+
```

Comment:
There is no need to fill the readme file with unrelated things like open tasks. GitHub has good tools to manage tasks.

Reply:
How would you do it? would the PR description itself be an option for this?

Next push I'll remove this change

Reply:
> How would you do it? would the PR description itself be an option for this?

Create an issue report or a pull request and then we can add it to a related project.

Reply:
ok, i'll probably create an issue for the coding styling update.